modules/oc-by-example-content.adoc

```diff
@@ -1857,10 +1857,39 @@ Log in to a server
   oc login localhost:8443 --web --callback-port 8280
 
   # Log in to the external OIDC issuer through Auth Code + PKCE by starting a local server listening port 8080
-  oc login localhost:8443 --exec-plugin=oc-oidc --client-id=client-id --extra-scopes=email,profile --callback-port=8080
+  oc login --exec-plugin=oc-oidc --issuer-url=<issuer_url> --client-id=<client_id> --extra-scopes=email,profile --callback-port=8080
+
+ifdef::openshift-dedicated,openshift-rosa[]
+  # Log in with an external OIDC if the external OIDC certificate is not publically trusted
+  oc login --exec-plugin=oc-oidc --issuer-url=<issuer_url> --client-id=<client_id> --extra-scopes=email --callback-port=8080 --oidc-certificate-authority <CA for external OIDC certificate>
+endif::openshift-dedicated,openshift-rosa[]  
 ----
 
+ifdef::openshift-dedicated,openshift-rosa[]
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--exec-plugin
+|Specify credentials exec plugin type to be used to authenticate external OIDC issuer. Currently only `oc-oidc` is supported.
+
+|--issuer-url
+|Issuer url for external issuer. Required.
+
+|--client-id
+|Client ID for external OIDC issuer. Only supports Auth Code + PKCE. Required.
+
+|--extra-scopes
+|Extra scopes for external OIDC issuer. Optional.
+
+|--callback-port
+|Port for the callback server when using --web. Defaults to a random open port.
```

Comment:
```suggestion
|Port for the callback server when using `--web`. The default value is any open port.
```

Reply:
This `--callback-port` flag can be used with the ealirer `--web` feature, also can be used with the `without oauth server` feature. This PR is for the latter. So, suggest as below to be more common wording.
```suggestion
|Port for the callback server redirected to after authentication flow is completed. Defaults to a random open port.
```